app/core/GasPolling/types.ts

```diff
@@ -153,37 +76,40 @@ export interface TransactionSharedProps {
   /**
    * For UpdateEIP1559Transaction, the transactionState are undefined.
    */
-  transactionState: {
-    assetType: string | undefined;
-    ensRecipient: string | undefined;
-    id: string | undefined;
-    nonce: string | undefined;
-    paymentRequest: string | undefined;
-    proposedNonce: string | undefined;
-    readableValue: string | undefined;
-    selectedAsset: Record<string, unknown>;
-    symbol: string | undefined;
-    transaction: {
-      data: string | undefined;
-      from: string | undefined;
-      gas: string | undefined;
-      gasPrice: string | undefined;
-      maxFeePerGas: string | undefined;
-      maxPriorityFeePerGas: string | undefined;
-      to: string | undefined;
-      value: string | undefined;
-    };
-    transactionFromName: string | undefined;
-    transactionTo: string | undefined;
-    transactionToName: string | undefined;
-    transactionValue: string | undefined;
-    type: string | undefined;
-    warningGasPriceHigh: string | undefined;
-  };
+  transactionState: TransactionStateProps;
   contractExchangeRates: Record<string, unknown>;
 }
 
-export interface GetEIP1559TransactionDataProps extends TransactionSharedProps {
+export interface UseGasTransactionProps extends TransactionSharedProps {
+  /**
+   * The selected gas value (low, medium, high). Gas value can be null when the advanced option is modified.
+   */
+  gasSelected: string | null;
+  onlyGas?: boolean;
+
```

Comment:
Can you change the format of the documentation to,
```suggestion
/**
 * @type UseGasTransactionProps
 *
 * <Definition>
 * @property gasSelected - <Definition>
 * @property onlyGas - <Definition>
 */
export interface UseGasTransactionProps extends TransactionSharedProps {
  gasSelected: string | null;
  onlyGas?: boolean;

```